ETC.tsv

```diff
@@ -5380,7 +5380,7 @@ ETC_20150317_005379	Good Job!
 ETC_20150317_005380	Where are you!!!
 ETC_20150317_005381	Come out!
 ETC_20150317_005382	Thorny Gateway
-ETC_20150317_005383	Bag that contains something
+ETC_20150317_005383	$Bag with Something Inside
```

Comment:
You can actually see the bag, and seeing that it says "Bag with Something Inside" is a bit silly, since you can tell that the bag contains something by just looking at it.  I'd suggest just "Bag".